_clients/k8s-operator.md

```diff
@@ -0,0 +1,146 @@
+
+
+## Installation 
+
+There are two ways to get started with the operator:
+
+- [Use a Helm chart](#use-a-helm-chartuse-a-helm).
+- [Use a local installation](#use-a-local-installation).
+
+### Use a Helm chart
+
+If you use Helm to manage your Kubernetes cluster, you can use the OpenSearch Kubernetes Operator's Cloud Native Computing Foundation (CNCF) project stored in Artifact Hub, a web-based application for finding, installing, and publishing CNCF packages. 
+
+To begin, log in to your Kubernetes cluster and add the Helm repository (repo) from [Artifact Hub](https://opster.github.io/opensearch-Kubernetes-operator/). 
+
+```
+helm repo add opensearch-operator https://opster.github.io/opensearch-k8s-operator/
+```
+
+Make sure that the repo is included in your Kubernetes cluster. 
+
+```
+helm repo list | grep opensearch
+```
+
+Both the `opensearch` and `opensearch-operator` repos appear in the list of repos.
+
+
+Install the manager that operates all of the OpenSearch Kubernetes Operator's actions. 
+
+```
+helm install opensearch-operator opensearch-operator/opensearch-operator
+```
+
+After the installation completes, the operator returns information on the deployment with `STATUS: deployed`. Then you can configure and start your [OpenSearch cluster](#deploy-a-new-opensearch-cluster).
+
+### Use a local installation
+
+If you want to create a new Kubernetes cluster on your existing machine, use a local installation. 
+
+If this is your first time running Kubernetes and you intend to run through these instructions on your laptop, make sure that you have the following installed: 
+
+- [Kubernetes](https://kubernetes.io/docs/tasks/tools/) 
+- [Docker](https://docs.docker.com/engine/install/)
+- [minikube](https://minikube.sigs.k8s.io/docs/start/)
+
+Before running through the installation steps, make sure that you have a Kubernetes environment running locally. When using minikube, open a new terminal window and enter `minikube start`. Kubernetes will now use a containerized minikube cluster with a namespace called `default`.
+
+Then install the OpenSearch Kubernetes Operator using the following steps:
+
+1. In your preferred directory, clone the [OpenSearch Kubernetes Operator repo](https://github.com/Opster/opensearch-k8s-operator). Navigate into repo's directory using `cd`.
+2. Go to the `opensearch-operator` folder.
+3. Enter `make build manifests`.
+4. Start a Kubernetes cluster. When using minikube, open a new terminal window and enter `minikube start`. Kubernetes will now use a containerized minikube cluster with a namespace called `default`. Make sure that `~/.kube/config` points to the cluster.
+
+```yml
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: /Users/naarcha/.minikube/ca.crt
```

Comment:
you shouldn't include real paths / usernames in the documentation, it provides no added value and just causes confusion. (same applies to the other 2 places where you have your username in the example file)